apps/web/package.json

```diff
@@ -2,12 +2,13 @@
   "name": "@gc-digital-talent/web",
   "private": true,
   "version": "1.0.0",
+  "type": "module",
   "license": "AGPL-3.0",
   "description": "",
   "scripts": {
-    "build": "webpack --config webpack.prod.js",
+    "build": "vite build --mode production --minify",
     "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf dist",
-    "dev": "webpack --config webpack.dev.js",
+    "dev": "vite build --sourcemap --mode=development",
```

Comment:
Maybe I never noticed this before because I always used watch mode but codgen isn't caching when I run this.

Reply:
Oh yeah, we disabled it because it seemed to be over-caching :cry: Maybe worth another pass to see if we can figure that out. 

https://github.com/GCTC-NTGC/gc-digital-talent/blob/acd32b7e1f97a0e0138a82374e8d8fbfd9caa176/turbo.json#L27